Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/mysql/MySQLMvpModel.java

```diff
@@ -73,9 +78,17 @@ public static Server findServer(final String subscriptionId, final String resour
     public static Server create(final String subscriptionId, final String resourceGroupName, final String serverName,
                                 Region region, final ServerPropertiesForDefaultCreate properties) {
         final MySQLManager manager = AuthMethodManager.getInstance().getMySQLManager(subscriptionId);
-        /*Sku sku = new Sku().withName("GP_Gen5_4");*/
+        List<PerformanceTierPropertiesInner> tiers = manager.locationBasedPerformanceTiers().inner().list(region.getName());
+        PerformanceTierPropertiesInner tier = tiers.stream().filter(e -> StringUtils.equals("Basic", e.id())).findFirst().orElse(
+            tiers.stream().filter(e -> StringUtils.equals("GeneralPurpose", e.id())).findFirst().orElse(
```

Comment:
can merge in one predicate function

Reply:
no. the sort of these predication is considered by the code.

Reply:
refactor